README.md

```diff
@@ -73,10 +73,10 @@ An object is identified as a built-in using:
 
 #### `[[Builtin]]` internal slot
 
-All built-ins, with the exception of the intrinsic object `%ObjectPrototype%`,
-would have a `[[Builtin]]` internal slot with a string value identifying the
-name of the built-in. For instance, for the `%Math%` intrinsic object, the
-value of `[[Builtin]]` is `'Math'`.
+All built-ins, with the exception of the intrinsic objects `%ObjectPrototype%`,
+and `%Object%` have a `[[Builtin]]` internal slot with a string value
```

Comment:
@jasnell "is" isn't checking the type, it's comparing the results of the internal slot, which is not necessarily the same. That said, I think built-in prototypes should only be special if they are special/exotic - meaning, `Function.prototype`, but not `String.prototype`.

Although, separately, should the single-argument `Builtin.is` only work if the Symbol.builtin method is unmodified? Filed as https://github.com/jasnell/proposal-istypes/issues/24